src/System.Runtime.Numerics/src/System/Numerics/BigCalc.SquMul.cs

```diff
@@ -0,0 +1,356 @@
+﻿using System.Diagnostics;
+using System.Security;
+
+namespace System.Numerics
+{
+    internal static partial class BigCalc
+    {
+#if DEBUG
+
+        private const int SquareThreshold = 128 / 32;
+        private const int MultiplyThreshold = 128 / 32;
```

Comment:
Per the [style guidelines](https://github.com/dotnet/corefx/wiki/Coding-style), I believe these are supposed to be `_multiplyThreshold` and `_squareThreshold`?